argocd-server/output.tf

```diff
@@ -0,0 +1,4 @@
+output "random_string" {
```

Comment:
also change output value to argocd password

Reply:
done